app/app.go

```diff
@@ -1,19 +1,20 @@
 package app
 
 import (
-	"cosmossdk.io/math"
 	"encoding/json"
 	"fmt"
-	"github.com/classic-terra/core/v3/app/mempool"
-	signer_extraction "github.com/skip-mev/block-sdk/adapters/signer_extraction_adapter"
-	"github.com/skip-mev/block-sdk/block"
-	blockbase "github.com/skip-mev/block-sdk/block/base"
 	"io"
 	stdlog "log"
 	"net/http"
 	"os"
 	"path/filepath"
 
+	"cosmossdk.io/math"
+	"github.com/classic-terra/core/v3/app/mempool"
+	signer_extraction "github.com/skip-mev/block-sdk/adapters/signer_extraction_adapter"
+	"github.com/skip-mev/block-sdk/block"
+	blockbase "github.com/skip-mev/block-sdk/block/base"
```

Comment:
i think because this branch is base off the `feat/unfork-cosmos-sdk` that @hoank101 and @TropicalDog17 is working on rightnow.  
@hoank101 @TropicalDog17 can you check this for me?